app/domain/dto/foundationInfo.dto.ts

```diff
@@ -0,0 +1,24 @@
+export type ContactInfoData = {
```

Comment:
```suggestion
export type ContactInfoDTO = {
```
I think it would be more understandable  
If we write SomeDTO naming for all DTOs